### PR TITLE
Save story metadata when scraping stories and media metadata flag is on

### LINKED ISSUE #347: Media Type not logged for Stories
Command:
`instagram-scraper -u myuser -p mypass --cookiejar cookies.txt -i --latest-stamps timestamps.txt --media-metadata -f users.txt --media-types story`

Expected:
Metadata for the stories should be logged in the json file

Output:
Stories are downloaded, but the metadata is not saved.

### PR DESCRIPTION
**Pull request by punchagan**
The metadata of the stories is saved to the metadata file along side the other metadata. Without this the media downloaded from stories is hard to work with. 

Closes #347

### CODE REVIEW

File: instagram_scraper/app.py [hunk 1]
```diff
@@ -845,6 +842,9 @@ def __fetch_stories(self, url):
 
                 for reel_media in retval['data']['reels_media']:
                     items.extend([self.set_story_url(item) for item in reel_media['items']])
+                    for item in reel_media['items']:
+                        item['highlight'] = fetching_highlights
```
Comment (AlexNik) [inline instagram_scraper/app.py]: Where do you use it?

File: instagram_scraper/app.py [hunk 2]
```diff
@@ -835,7 +832,7 @@ def get_shared_data(self, username=''):
             except (TypeError, KeyError, IndexError):
                 pass
 
-    def __fetch_stories(self, url):
+    def __fetch_stories(self, url, fetching_highlights=False):
```
Comment (AlexNik) [inline instagram_scraper/app.py]: Why do you choose `fetching_highlights` name?
Mb `fetching_highlights_metadata`?

File: instagram_scraper/app.py (hunk 1 above)
Comment (punchagan) [reply instagram_scraper/app.py]: I use it for post processing, not in the scraper.

File: instagram_scraper/app.py (hunk 2 above)
Comment (punchagan) [reply instagram_scraper/app.py]: Fixed this.

**Pull request merged**